src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -257,6 +257,7 @@ export const ArtworkSidebarCommercialButtons: React.FC<ArtworkSidebarCommercialB
             input: {
               artworkId: artwork.internalID,
               editionSetId: selectedEditionSet?.internalID,
+              partnerOfferId: activePartnerOffer?.internalID,
```

Comment:
Maybe we can hide this change behind the same `partnerOfferVisibilityEnabled` feature flag?

```suggestion
              ...(partnerOfferVisibilityEnabled ? {partnerOfferId: activePartnerOffer?.internalID} : {})
```

Reply:
nvm, just checked that activePartnerOffer is hidden behind the feature flag :)

Reply:
activePartnerOffer is already behind the feature flag (line 67) so we don't need to use it in here